x-pack/plugins/security/server/licensing/license_service.ts

```diff
@@ -69,7 +69,7 @@ export class SecurityLicenseService {
             };
           }
 
-          const isLicensePlatinumOrTrial = rawLicense.isOneOf(['platinum', 'trial']);
+          const isLicensePlatinumOrTrial = rawLicense.isOneOf(['platinum', 'enterprise', 'trial']);
```

Comment:
Naming this is hard...what do you think about something like this?

```suggestion
          const isLicensePlatinumOrBetter = rawLicense.isOneOf(['platinum', 'enterprise', 'trial']);
```